controlplane/hack/deploy/addon/policy/hub/clustermanagementaddon/clustermanagementaddon.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
```

Comment:
Do you also have the `governance-policy-framework` `ClusterManagementAddOn` manifest?

Reply:
we only add config-policy-addon for test at the first stage.